test/sql/timestamp.sql

```diff
@@ -362,7 +362,7 @@ FROM unnest(ARRAY[
 \set ON_ERROR_STOP 0
 SELECT time_bucket(10::smallint, '-32768'::smallint);
 SELECT time_bucket(10::smallint, '-32761'::smallint);
-select time_bucket(10::smallint, '-32000'::smallint, 1000::smallint);
+select time_bucket(10::smallint, '-32768'::smallint, 1000::smallint);
```

Comment:
In addition to these can you test what happens when offset is `INT_MIN` and `INT_MAX`

Reply:
done